webrtc-stats.html

```diff
@@ -1741,6 +1743,19 @@ <h2>
                   other than 0, the round trip time is left undefined.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>totalRoundTripTime</code></dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the cumulative sum of all round trip time measurements in seconds 
+                  since the beginning of the session. The individual round trip time is calculated 
+                  based on the RTCP timestamps in the RTCP Receiver Report (RR) [[!RFC3550]], hence
+                  undefined roundtrip are not added. The average round trip time can be computed 
+                  from <code>totalRoundTripTime</code> by  dividing it by <code>reportsReceived</code>.
```

Comment:
I spoke too soon: You've defined sent/received as "contain a valid round trip time", so my question now is, does it make sense for a metric called "reportsSent/reportsReceived" to not increase if we get a report because something about that report does not fulfill a criteria? Should we rename the metric or have multiple metrics?

Why would timestamp not be present?